pycytominer/tests/test_cyto_utils/test_cells.py

```diff
@@ -265,15 +278,15 @@ def test_SingleCells_count():
 
 
 def test_load_compartment():
-    loaded_compartment_df = AP.load_compartment(compartment="cells")
+    loaded_compartment_df = AP.load_compartment(compartment="cells", features="infer")
```

Comment:
if you add `features="infer"` as default, you can probably delete this explicit variable naming to retain default testing configuration